packages/now-build-utils/src/detect-builder.ts

```diff
@@ -76,6 +76,12 @@ export function ignoreApiFilter(file: string) {
     return false;
   }
 
+  // If the file does not match any builder we also
+  // don't want to create a reoute e.g. `package.json`
+  if (API_BUILDERS.some(({ src }) => minimatch(file, src)) === false) {
```

Comment:
```suggestion
  if (API_BUILDERS.every(({ src }) => !minimatch(file, src))) {
```

I think you can invert this